subjects/weareunique/README.md

```diff
@@ -0,0 +1,60 @@
+## Weareunique
+
+### Instructions
+
+Write a function that takes two strings and returns the number of characters that are not included in both, without doubles.
+- If there is no unique charachters return 0.
+- If both strings are empty  or one of them return -1.
+
+### Expected function
+
+```go
+
+func Weareunique(str1 , str2 string) int {
+
+}
+
+```
+
+### Usage
+
+Here is a possible program to test your function :
+
+```go
+
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+
+	arr := [][]string{
+		{"foo", "boo"},
+		{"",""},
+		{"abc", "def"},
+		{"hello", "yoall"},
+		{"everyone", ""},
```

Comment:
Your solution returns 6 here

Reply:
I forgot to push the right solution in  the test repository but now it's good